src/pages/Assets/Assets.tsx

```diff
@@ -7,7 +7,9 @@ import { Page } from 'components/Layout/Page'
 export const Assets = () => {
   const history = useHistory()
   const onClick = (asset: Asset) => {
-    const url = `/assets/${encodeURIComponent(asset.caip19)}`
+    // CAIP19 has a `/` separator so the router will have to parse 2 variables
+    // e.g., /assets/:chainId/:assetSubId
+    const url = `/assets/${asset.caip19}`
```

Comment:
Would it make sense to deserialize it in place instead of a comment?

Reply:
> Would it make sense to deserialize it in place instead of a comment?

You mean like `asset.caip19.split('/')` then join it again? I'd rather just add a comment than do unnecessary work.